srcpkgs/grabc/template

```diff
@@ -1,20 +1,26 @@
 # Template file for 'grabc'
 pkgname=grabc
-version=1.1
+reverts=1.1_1
+version=1.0.2
```

Comment:
The version change was weird to me last I looked.

I'd like @Duncaen to take a look.

Reply:
Judging from both the version string in the original tarball & the changelog in the newer README, upstream decided to change the version scheme.